lib/shopify-cli/messages/messages.rb

```diff
@@ -328,6 +328,14 @@ module Messages
             {{underline:https://shopify.github.io/shopify-app-cli/migrate/}}
 
           MESSAGE
+
+          new_version: <<~MESSAGE,
+          {{*}} {{yellow:A new version of the Shopify App CLI is available! You have version %s and the latest version is %s.
+
+            To upgrade, follow the instructions for the package manager you’re using:
+            {{underline:https://shopify.github.io/shopify-app-cli/getting-started/upgrade/}}}}
```

Comment:
Maybe future functionality: do we know what package manager they're using? Can we just give them the right command?

Reply:
Yeah, I thought about that too - it's a bit tricky given that even the `apt` or `yum` installers actually use `gem install` inside them (i.e., they're basically just wrappers for `gem install`).  So, probably doable, but not trivial.